client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -48,8 +48,9 @@ class TransferDomainPrecheck extends React.PureComponent {
 
 	onClick = () => {
 		const { domain, supportsPrivacy } = this.props;
+		const { registrar, registrarIanaId } = this.state;
```

Comment:
Those should be `losingRegistrar` and `losingRegistrarIanaId` :) It's not working right now

Reply:
Ah...sorry. Fixed.